src/java.base/share/classes/java/io/File.java

```diff
@@ -82,7 +82,9 @@
  *
  * <p> Unless otherwise noted, {@linkplain java.nio.file##links symbolic links}
  * are automatically redirected to the <i>target</i> of the link, whether they
- * are provided by a pathname string or via a {@code File} object.
+ * are provided by a pathname string or via a {@code File} object.  Methods
```

Comment:
```suggestion
 * are provided by a pathname string or via a {@code File} object. Methods
```

Reply:
You will notice that in this source file the convention is to separate with two spaces the period ending a sentence with the first letter of the subsequent sentence. In any case, it makes no difference in the rendered HTML.

Reply:
I see, thanks for the clarification 👍🏼